src/test/java/iudx/resource/server/database/async/AsyncServiceTest.java

```diff
@@ -419,7 +419,7 @@ public void testScrollQueryWithInvalidQuery(VertxTestContext vertxTestContext) {
     ProgressListener progressListener = mock(ProgressListener.class);
     asyncService2 = new AsyncServiceImpl(Vertx.vertx(), client, postgresService, fileOpsHelper, filePath);
     when(jsonObject.put(anyString(), anyBoolean())).thenReturn(jsonObject);
-    asyncService2.scrollQuery(file, jsonObject, "Dummy SearchID", progressListener, handler -> {
+    asyncService2.scrollQuery(file, jsonObject, "Dummy SearchID", progressListener, "csv", handler -> {
```

Comment:
Why hardcoded? format should be extracted from the consumed RMQ message.

Reply:
We might not get the RMQ message while testing each method in the AsyncServiceTest class